public/js/custom.js

```diff
@@ -306,12 +306,12 @@ var Samar = function(){
 				var menu = jQuery('.sticky-header');
 				if ($(window).scrollTop() > menu.offset().top) {
 					menu.addClass('is-fixed');
-					$('.site-header .container > .logo-header .logo').attr('src','images/logo.png');
-					$('.site-header .container > .logo-header .logo-2').attr('src','images/logo-2.png');
-					$('.site-header .container > .logo-header .logo-3').attr('src','images/logo-3.png');
+					$('.site-header .container > .logo-header .logo').attr('src','images/medha.png');
+					$('.site-header .container > .logo-header .logo-2').attr('src','images/medha.png');
+					$('.site-header .container > .logo-header .logo-3').attr('src','images/medha.png');
 				} else {
 					menu.removeClass('is-fixed');
-					$('.site-header .container > .logo-header .logo, .site-header .container > .logo-header .logo-2, .site-header .container > .logo-header .logo-3').attr('src','images/logo-white.png')
+					$('.site-header .container > .logo-header .logo, .site-header .container > .logo-header .logo-2, .site-header .container > .logo-header .logo-3').attr('src','images/medha.png')
```

Comment:
this should be white logo to appear on color theme

Reply:
please use this image: https://drive.google.com/file/d/1xnmXpwdm_xl9j3erEY7zT0uWDuLq1HIw/view?usp=sharing

Reply:
done

Reply:
i have again done it